tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1867,6 +1872,8 @@ def rgb_to_grayscale(images, name=None):
   last dimension of the output is 1, containing the Grayscale value of the
   pixels.
 
+  Sample usage with_image can be found [here on Github](https://github.com/Mbah-Javis/How-to-convert-RGB-image-to-grayscale/blob/master/RBG_to_GrayScale_using_tf_image_rgb_to_grayscale.ipynb) or [here](https://colab.research.google.com/drive/1VFbIMRKGWSknlQ_Z_d1ihqSDW08Vu117)
```

Comment:
No links to random pages on github we can't ensure stay up-to-date

Reply:
Okay. Can I submit it to tensorflow-docs tutorials that way up to date will be ensured right?
Am a participant in GCI